test/zdtm.py

```diff
@@ -283,6 +283,14 @@ def init(self, l_bins, x_bins):
                     self.__construct_root()
                     os.mknod(self.root + "/.constructed", stat.S_IFREG | 0o600)
 
+        try:
+            with open(self.root + "/dev/null"):
+                pass
+        except IOError as e:
+            if e.errno != errno.EACCES:
+                raise e
+            raise test_fail_exc("The filesystem of CRIU repo should not have 'nodev' flag. Consider putting CRIU on appropriate file system to run ZDTM.")
```

Comment:
It might be good to add `\n`:
```Suggestion
            raise test_fail_exc("\nThe filesystem of CRIU repo should not have 'nodev' flag. Consider putting CRIU on appropriate file system to run ZDTM.")
```

```
[root@thinkpad:/tmp/criu/test] <base> 1 # python3 ./zdtm.py run -t zdtm/static/busyloop00 -f ns
userns is supported
=== Run 1/1 ================ zdtm/static/busyloop00
======================= Run zdtm/static/busyloop00 in ns =======================
 Test zdtm/static/busyloop00 FAIL at 
The filesystem of CRIU repo should not have 'nodev' flag. Consider putting CRIU on appropriate file system to run ZDTM. 
##################################### FAIL #####################################
```